plugins/modules/win_psrepository_copy.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Brian Scholer <@briantist>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r'''
+---
+module: win_psrepository_copy
```

Comment:
You want to have `version_added: 1.3.0` to indicate when this module was added to the collection.

Reply:
haha, how easily I catch this on someone _else's_ PR.. will add!